src/collections/_documentation/clients/cocoa/index.md

```diff
@@ -43,6 +47,14 @@ Run `carthage update` to download the framework and drag the built _Sentry.frame
 
 We also provide a pre-built version for every release which can be downloaded at [releases on GitHub](https://github.com/getsentry/sentry-cocoa/releases).
 
+### SPM | Swift Package Manage
```

Comment:
Swift Package Manager

(So the anchor is clean)

Reply:
true